parts/linux/cloud-init/artifacts/cse_install_mariner.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+echo "Sourcing cse_install_distro.sh for Mariner"
+
+removeMoby() {
+    retrycmd_if_failure 10 5 60 dnf remove -y moby-engine moby-cli
+}
+
+removeContainerd() {
+    retrycmd_if_failure 10 5 60 dnf remove -y moby-containerd
+}
+
+installDeps() {
+    apt_get_update || exit $ERR_APT_UPDATE_TIMEOUT
+    apt_get_dist_upgrade || exit $ERR_APT_DIST_UPGRADE_TIMEOUT
+    for apt_package in ca-certificates cifs-utils cracklib ebtables ethtool fuse git iotop iproute ipset iptables jq pam nfs-utils socat sysstat traceroute util-linux xz zip; do
+      if ! apt_get_install 30 1 600 $apt_package; then
+        exit $ERR_APT_INSTALL_TIMEOUT
+      fi
+    done
+    if [[ "${AUDITD_ENABLED}" == true ]]; then
+      if ! apt_get_install 30 1 600 audit; then
+        exit $ERR_APT_INSTALL_TIMEOUT
+      fi
+    fi
+}
+
+installGPUDrivers() {
+    echo "GPU drivers not yet supported for Mariner"
+    exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
+}
+
+installSGXDrivers() {
+    echo "SGX drivers not yet supported for Mariner"
+    exit $ERR_SGX_DRIVERS_START_FAIL
+}
+
+getMobyPkg() {
+    echo "Moby is available in the Mariner package repository"
+}
+
+{{- if NeedsContainerd}}
```

Comment:
I think for Mariner we only want/need to support containerd so we don't need to worry about docker